### ISSUE TITLE
Why does `cov` does not take a `mean` keyword argument?

### ISSUE THREAD

**Issue by theogf** [JuliaStats/StatsBase.jl#751]
It looks like the underlying `covm` and co are using it and this would possibly save some computation time for the `mean_and_cov` implementation.

**Comment by theogf** [comment]
Right, so you just followed their given API. I will make an issue in the Julia repo then.

**Comment by nalimilan** [comment]
Note that Statistics is in its own repo now: https://github.com/JuliaLang/Statistics.jl
